plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -489,6 +489,13 @@ func ClusterRoles() []rbacv1.ClusterRole {
 				rbacv1helpers.NewRule("create").Groups(certificatesGroup).Resources("certificatesigningrequests/selfnodeclient").RuleOrDie(),
 			},
 		},
+		{
+			// a role making azure cloud provider to send events
+			ObjectMeta: metav1.ObjectMeta{Name: "system:azure-cloud-provider"},
```

Comment:
This is the role that @liggitt noted shouldn't go into core.

Reply:
@deads2k Thanks. Then let me remove this and add a release note on it. This is a user-fencing change.